### PR TITLE
Fix docstring issues leading to sphinx errors in generate_corsika_histograms

### LINKED ISSUE #670: Add generate_corsika_histograms.py back to documentation
sphinx fails when adding generate_corsika_histograms.py to docs/source/applications.rst with an error in ctapipe (!):

```
eading sources... [100%] visualization

Warning, treated as error:
autodoc: failed to import module 'generate_corsika_histograms'; the following exception was raised:
Traceback (most recent call last):
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/sphinx/ext/autodoc/importer.py", line 58, in import_module
    return importlib.import_module(modname)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/maierg/DESYCloud/CTA/Pipelines/MC-Pipeline/gammasim-tools/simtools/simtools/applications/generate_corsika_histograms.py", line 176, in <module>
    from simtools.corsika.corsika_histograms import CorsikaHistograms
  File "/Users/maierg/DESYCloud/CTA/Pipelines/MC-Pipeline/gammasim-tools/simtools/simtools/corsika/corsika_histograms.py", line 15, in <module>
    from ctapipe.io import read_table, write_table
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/io/__init__.py", line 7, in <module>
    from .astropy_helpers import read_table, write_table
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/io/astropy_helpers.py", line 13, in <module>
    from .hdf5tableio import (
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/io/hdf5tableio.py", line 13, in <module>
    from .tableio import (
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/io/tableio.py", line 13, in <module>
    from ..instrument import SubarrayDescription
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/instrument/__init__.py", line 1, in <module>
    from .atmosphere import get_atmosphere_profile_functions
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/instrument/atmosphere.py", line 8, in <module>
    from ..utils import get_table_dataset
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/utils/__init__.py", line 14, in <module>
    from .event_type_filter import EventTypeFilter
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/utils/event_type_filter.py", line 1, in <module>
    from ..containers import EventType
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/containers.py", line 264, in <module>
    class CameraHillasParametersContainer(BaseHillasParametersContainer):
  File "/Users/maierg/miniconda3/envs/simtools-dev-311/lib/python3.11/site-packages/ctapipe/containers.py", line 271, in CameraHillasParametersContainer
    x = Field(nan * u.m, "centroid x coordinate", unit=u.m)
              ~~~~^~~~~
TypeError: unsupported operand type(s) for *: 'float' and 'm'

make: *** [html] Error 2
```

### PR DESCRIPTION
**Pull request by GernotMaier**
closes #670

### CODE REVIEW

**Review by VictorBarbosaMartins — APPROVED** [review]
I tested locally and it works perfectly. Thank you!

File: simtools/applications/generate_corsika_histograms.py [hunk 1]
```diff
@@ -42,68 +42,70 @@
...
+
+    .. code-block:: console
+
+        hist_direction:
+            x axis: {bins: 100, scale: linear, start: -1, stop: 1}
+            y axis: {bins: 100, scale: linear, start: -1, stop: 1}
+
+        hist_position:
+            x axis:
+            bins: 100
```
Comment (orelgueta) [inline simtools/applications/generate_corsika_histograms.py:65]: The indentation here is now wrong (it was right before).

Comment (GernotMaier) [reply simtools/applications/generate_corsika_histograms.py:65]: Correct - the indentation was generally wrong before, but now it is inconsistent with what we want to say.

Comment (VictorBarbosaMartins) [reply simtools/applications/generate_corsika_histograms.py:65]: @orelgueta, in the generated document it looks good.
Screenshot 2023-11-30 at 14 58 51

Comment (VictorBarbosaMartins) [reply simtools/applications/generate_corsika_histograms.py:65]: Ah, I see the tabs missing now :)

**Pull request merged**